client/java/src/main/java/com/linecorp/centraldogma/client/updater/CentralDogmaBeanFactory.java

```diff
@@ -56,13 +58,26 @@
  * > public class Foo {
  * >     private int a;
  * >     private String b;
+ * >     private Revision rev;
  * >
  * >     public int getA() { return a; }
  * >     public void setA(int a) { this.a = a; }
  * >     public String getB() { return b; }
  * >     public void setB(String b) { this.b = b; }
+ * >     public Revision getRevision() { return this.rev; }
```

Comment:
The field `rev` isn't used at all actually. We could just return null here, add some comment that this method is implemented by the factory and remove the field `rev`.